fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -210,22 +210,22 @@
-		"On a very quiet day, the Russian-famous Fiddler Diner was serving food to the crew, when they realized they ran out of burger ingredients. \
-		After drawing straws, the Cook was sent to fetch some more meat from the Morgue, unaware of the events that will transpire. \
-		'It's normal for the Kitchen to grab dead bodies, right? It's not like they need them... Right?' The Cook thought, \
-		inattentively grabbing the first body they could find, trying to get this over with before it becomes a memory. \
-		What the Cook hadn't noticed, the Morgue's tray was green, the body was filled with a soul, one that was begging not to be gibbed.",
-
-		"The Cook one'd and two'd the body into the gibber and turned it on, the grinder struggling to keep up on its unupgraded parts. \
-		Once the whole body entered the machine, it suddenly stopped working, and instead started spitting the meat back out, as if it was in reverse... \
-		The Cook looked over to see what was going on, but the slab of meat looked back, with massive eyes.",
-
-		"After a quick confiscation from the Russian Sol Government, most records of 'Beefmen' have gone dark, \
-		with minor glimpses of 'Sleep Experiments' going around. \
-		One thing is certain though, a rise of gibbers on-board Russian stations, which was followed by a rise in Beefmen. \
-		No one knows what happened during this era, but when the program finally ended and they were allowed into society, \
-		none of them were able to live 'normally'. Their complete inability to sleep, but their power in traversing the 'Phobetor Tear' \
-		immediately started popping everywhere, and has been at the	forefront of galaxy-wide investigations of their anatomy and the experiments.",
+		"On a very quiet day the Russian \"Fiddler Diner\" was serving food to its station's crew. Shortly into the lunch rush they ran out of burger meat. \
+		After drawing straws, a neophyte cook— blissfully unaware of what would soon transpire— was sent to fetch some meat from the morgue. \
+		The cook thought, \"It's normal for kitchens to repurpose dead bodies, right? It's not like the dead need them... right?\" \
+		Inattentively grabbing the first body they could find, the cook rushed to complete their unenviable task lest it were to become an enduring memory. \
+		What the cook hadn't noticed was the green revivability indicator on the morgue tray they were ransacking. The body had a soul— one that was determined to not have its vessal gibbed.",
+
+		"The young cook returned to their diner's cold room and shoved the corpse into the gibber. The machine struggled in tedium with its unupgraded parts. \
+		Once the whole body was in, the grinder's blades suddenly stopped and, with a horrible shriek, they began spinning in reverse. \
+		The cook looked over on impulse, but to their horror a grinning slab of meat looked back at them with somnambulant eyes.",
+
+		"A quick cover story was released by the Russian Sol Government, but it failed completely. \
```

Comment:
Failed completely is a massive change from the previous "quickly covered up with only minor stories"

Reply:
How could beefmen both become a species seen during the course of a shift on SS13 and have their true origins covered up? The beefmen are insane, yes, but I'm pretty sure they would remember how they were created.

Reply:
it is perfectly reasonable to not expect beefmen to know where they originate from and that was basically the expectation. they are sleeper experiments and even if they recovered some sense of sanity they are still completely insane.

Reply:
Is there a major source of beefman lore other than the text here that I'm unaware of? If so then I could look at that and make better adjustments here.

In any case, the current revisions that I'm suggesting don't necessarily conflict with the idea that Beefmen were creepypasta-esque Russian experiments— these suggestions just don't state that idea outright.

Reply:
They're Russian sleeper agents who are skinned alive and are now insane, which is why they can't sleep, can teleport, and had hallucinations before it was removed for being too invasive to the player. The current lore iteration I wrote and the community voted to make it the official one.